Sources/PointFree/Episode/Show.swift

```diff
@@ -721,6 +721,7 @@ let markdownBlockStyles: Stylesheet =
       <> (a & .pseudo(.link)) % color(Colors.purple150)
       <> (a & .pseudo(.visited)) % color(Colors.purple150)
       <> (a & .pseudo(.hover)) % color(Colors.black)
+      <> (p & .pseudo(.not(.pseudo(.lastChild)))) % margin(bottom: .rem(1.5))
```

Comment:
I only noticed this cause of the blog, but our paragraphs feel a bit bunched. adding this to space em out. this does cause a bunch of html snapshot failures (cause of the extra css), and a few episode screenshots (cause of the extra space).